yarn.lock

```diff
@@ -9528,6 +9535,7 @@ vscode-chokidar@microsoft/chokidar#42800a1d731853bd2cb3140e42a11d2f0ac5f436:
   version "1.6.5"
   resolved "https://codeload.github.com/microsoft/chokidar/tar.gz/42800a1d731853bd2cb3140e42a11d2f0ac5f436"
   dependencies:
+    anymatch "^2.0.0"
```

Comment:
Why were `anymatch` and `fsevents` changed ?

Reply:
@deepak1556 I ran `yarn` and the outcome was this change, not sure? I will try to get your change for `vscode-chokidar` in today, publish a new version and then fix this in the branch